projects/kubernetes-sigs/image-builder/BOTTLEROCKET_RELEASES

```diff
@@ -16,4 +16,4 @@
     raw-release-version: v1.19.1
 1-29:
     ami-release-version: v1.19.1
-    ova-release-version: 1.19.2
+    ova-release-version: v1.19.1
```

Comment:
Curious, why was this `1.19.2 `before?

Reply:
BR team gave us a dev version of the OVA stored in Cloudfront. They had originally planned to release the 1.29 OVA in 1.19.2 so they had named it accordingly in Cloudfront. But today they released the prod one to their 1.19.1 release endpoint so we're using that.